docs/stencil-docs/installing-stencil-cli/installing-stencil.md

```diff
@@ -48,29 +48,41 @@ nvm use 12
 # Install Stencil CLI
 npm install -g @bigcommerce/stencil-cli
 ```
+
+### Installing on Mac with Apple silicon
+
+Installing Stencil CLI and its dependencies on Macs that use Apple silicon, such as the M1 chip, requires Rosetta. Rosetta allows a Mac with Apple silicon to use apps built for a Mac with an Intel processor. The following steps will guide you through the installation process:
+
+
 <div class="HubBlock--callout">
 <div class="CalloutBlock--info">
 <div class="HubBlock-content">
 
 <!-- theme: info -->
 
-> Use these instructions for Mac's with a M1 chip.
+> These instructions have been tested on a **MacBook Air** with an M1 chip.
+
 
 </div>
 </div>
 </div>
 
-```shell
 
-# Install Node Version Manager (nvm)
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
+Open terminal using Rosetta.
+1. Open **Finder**.
+2. Go to **Applications** > **Utilities** > **Terminal**.
+3. Right-click **Terminal** and select **Get Info**.
+4. Check the **Open using Rosetta** checkbox.
+5. Close the window and quit all terminal instances.
+6. Start a new terminal, and install Rosetta if prompted.
 
+Run the following commands.

```

Comment:
Is this also a step in the process? Should it be numbered?

Reply:
This process is consistent with the other ones.

Reply:
The steps are consistent with the commands for the other platforms (Windows, Linux).